### PR TITLE
SSCS-9903: Location data – Populate location data on creation SYA and case update

### PR DESCRIPTION
**Pull request by rasikasri**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+<<<<<<< HEAD
```
Comment (jackmaloney) [inline benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json:2]: This doesn't look right

File: benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json [hunk 2]
```diff
@@ -1222,6 +1223,6 @@
   {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseFieldID": "caseAccessCategory", "AccessProfile": "citizen", "CRUD": "CRUD"},
   {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseFieldID": "showOtherPartyDetails", "AccessProfile": "caseworker-sscs", "CRUD": "CRUD"},
   {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseFieldID": "caseName", "AccessProfile": "caseworker-sscs", "CRUD": "CRUD"},
-  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseFieldID": "caseName", "AccessProfile": "citizen", "CRUD": "CRUD"}
-
+  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseFieldID": "caseName", "AccessProfile": "citizen", "CRUD": "CRUD"},
+  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "CaseFieldID": "caseManagementLocation", "UserRole": "caseworker-sscs", "CRUD": "CRUD"}
```
Comment (jackmaloney) [inline benefit/data/sheets/AuthorisationCaseField/AuthorisationCaseField.json:1227]: See Chris's comment in channel - this should not be UserRole, it should be 'AccessProfile'

File: benefit/data/sheets/ComplexTypes/ComplexTypes.json [hunk 3]
```diff
@@ -452,5 +452,7 @@
...
   {"LiveFrom": "18/12/2021", "ID": "OtherPartyOption", "ListElementCode": "otherPartyOptionName", "FieldType": "Text", "ElementLabel": "Other party name", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "18/12/2021", "ID": "OtherPartyOption", "ListElementCode": "otherPartyOptionId", "FieldType": "Text", "ElementLabel": "Other party id", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "18/12/2021", "ID": "OtherPartyOption", "ListElementCode": "resendToOtherParty", "FieldType": "YesOrNo", "ElementLabel": "Reissue", "SecurityClassification": "PUBLIC"}
+  {"LiveFrom": "18/12/2021", "ID": "OtherPartyOption", "ListElementCode": "resendToOtherParty", "FieldType": "YesOrNo", "ElementLabel": "Reissue", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "18/12/2021", "ID": "CaseManagementLocation", "ListElementCode": "region", "FieldType": "Text", "ElementLabel": "Region", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "18/12/2021", "ID": "CaseManagementLocation", "ListElementCode": "baseLocation", "FieldType": "Text", "ElementLabel": "Base Location", "SecurityClassification": "PUBLIC"}
```
Comment (jackmaloney) [inline benefit/data/sheets/ComplexTypes/ComplexTypes.json:457]: is uppercase C the correct thing for the id here?

Comment (rasikasri) [reply benefit/data/sheets/ComplexTypes/ComplexTypes.json:457]: I see complex types have started with both uppercase letters as well as lowercase. I think upper case is more suitable as this is a new type. Anyway no any functional issues with this ID

**Review by jackmaloney — APPROVED** [review]

**Pull request merged**